apps/passport-server/src/services/semaphoreService.ts

```diff
@@ -89,7 +89,6 @@ export class SemaphoreService {
       logger(`[SEMA] Reloading semaphore service...`);
 
       await this.reloadZuzaluGroups();
-      await this.reloadGenericGroup();
```

Comment:
should we add a "turned off for devconnect" comment here too?

Reply:
sure